.github/workflows/build.yml

```diff
@@ -32,6 +27,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      # Only needed for arm, but seems to auto-disable on x86:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```

Comment:
Don't we (can't we) have access to a real arm64 machine?

Reply:
> Unfortunately, it runs podman and alma9, and that's incompatible with the container build actions that are used in the CI of this repo.

What makes it incompatible? :(

Reply:
podman doesn't have `docker buildx`.

Reply:
More precisely, buildx is an alias to `podman build`, and this doesn't support most of the arguments of `docker buildx`

Reply:
There is another advantage: I'm building now a multi-arch image, so we don't have to change anything downstream.
This is only easy if this is done in a single run, so I think we might stay with QEMU for a long time.

Reply:
The respective comment is being added.